### ISSUE TITLE
bot bench: Allow passing feature flags

### ISSUE THREAD

**Issue by xermicus** [paritytech/command-bot#288]
`pallet-revive` requires the `riscv` feature for benchmarks. We don't want to add it to `runtime-benchmarks` as that will require everybody else to have the rve toolchain installed. So we'd need a way to pass flags to the bot, however this doesn't seem to be supported currently? I imagine something like this should work: https://github.com/paritytech/polkadot-sdk/pull/5675#issuecomment-2343547579

**Comment by mordamax** [comment]
Not a problem to add, though we're going to deprecate this command bot along with old benchmarks, now we have the other command bot built as [GHA](https://github.com/paritytech/polkadot-sdk/pull/5457), and it uses frame-omni-bencher. 
I suspect this still will require riscv feature to be added in your case?

**Comment by xermicus** [comment]
Yeah we need the rve toolchain and a way to set the feature flag in the omni bencher and we should be good. Thanks!
